package.json

```diff
@@ -2,13 +2,73 @@
   "name": "link-shortener",
   "version": "1.0.0",
   "main": "index.js",
-  "author": "Valera <Koniushok@users.noreply.github.com>",
+  "author": {
+    "name": "Valery Koniushok",
+    "email": "Kokoniushok2000@gmail.com",
+    "url": "https://github.com/Koniushok"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Koniushok/link-shortener"
+  },
+  "description": "link shortening service",
+  "license": "UNLICENSED",
   "private": true,
+  "bugs": {
+    "url": "https://github.com/Koniushok/link-shortener/issues",
+    "email": "koniushok2000@gmail.com"
+  },
   "dependencies": {
-    "webpack": "^4.28.3",
-    "webpack-cli": "^3.2.1"
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "react-hot-loader": "^4.6.3",
+    "styled-components": "^4.1.3"
   },
   "scripts": {
-    "build": "webpack --mode production"
+    "start": "webpack-dev-server --mode development --open",
+    "start-hot": "webpack-dev-server --mode development --open --hot",
```

Comment:
Combine please start and star hot commands. You'll always use only 1 comment with hot